__tests__/index.test.js

```diff
@@ -38,3 +41,38 @@ describe("1. GET /api/topics", () => {
       });
   });
 });
+
+describe("2. GET /api/articles/:article_id", () => {
+  test("status:200 sends back an article  object with the following properties", () => {
+    const article_titles = "Living in the shadow of a great man";
+    const art_id = 3;
+    return request(app)
+      .get(`/api/articles/${art_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(typeof articles).toBe("object");
+
+        expect(articles).toEqual(
+          expect.objectContaining({
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+          })
+        );
+      });
+  });
+
+  test("GET:400 responds with an appropriate error message when provided with a incorrect article_id submitted)", () => {
+    return request(app)
+      .get("/api/articles/apples")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad Request");
+      });
+  });
+});
```

Comment:
Need to test for what happens if we make a valid request but the article doesn't exist eg `/api/artilces/9999`